src/redux/reducer.js

```diff
@@ -59,15 +61,47 @@ function loadApi(state = initialState.get('loadApi'), action) {
 
 function REPL(state = initialState.get('REPL'), action) {
   switch (action.type) {
+    case actType.GET_PREV_REPL_CODE:
+      //if codeIndex hasn't reached the begining of history and
+      //the previous code isn't an empty string
+      if (state.get('codeIndex') < state.get('history').size &&
```

Comment:
I ended up renaming it to REPLcodePosition and added a comment next to it clarifying it best I could